gnutls/Interoperability/TLSv1-2-with-NSS/runtest.sh

```diff
@@ -93,27 +93,33 @@ rlJournalStart
         # RSA key exchange ciphers
         #
         if ! fipsIsEnabled; then
-            C_NAME[$i]="TLS_RSA_WITH_RC4_128_MD5"
-            C_OPENSSL[$i]="RC4-MD5"
-            C_ID[$i]="0004"
-            C_TLS1_2_ONLY[$i]="False"
-            C_SUBCA[$i]="$(x509Cert rsa-ca)"
-            C_CERT[$i]="$(x509Cert rsa-server)"
-            C_KEY[$i]="$(x509Key rsa-server)"
-            C_CLNT_CERT[$i]="$(x509Cert rsa-client)"
-            C_CLNT_KEY[$i]="$(x509Key rsa-client)"
-            i=$(($i+1))
-
-            C_NAME[$i]="TLS_RSA_WITH_RC4_128_SHA"
-            C_OPENSSL[$i]="RC4-SHA"
-            C_ID[$i]="0005"
-            C_TLS1_2_ONLY[$i]="False"
-            C_SUBCA[$i]="$(x509Cert rsa-ca)"
-            C_CERT[$i]="$(x509Cert rsa-server)"
-            C_KEY[$i]="$(x509Key rsa-server)"
-            C_CLNT_CERT[$i]="$(x509Cert rsa-client)"
-            C_CLNT_KEY[$i]="$(x509Key rsa-client)"
-            i=$(($i+1))
+            #
+            # Crypto Policy handling seems to be broken in NSS on F25 for RC4
+            # BZ#1426267
```

Comment:
Fair point. This can be easily done by enforcing NSS to ignore the crypto policies entirely by setting the `NSS_IGNORE_SYSTEM_POLICY` env variable to 1.